apps/fluent-tester/src/TestComponents/RadioGroupV1/CustomizedRadioGroup.tsx

```diff
@@ -27,7 +27,7 @@ export const CustomizedRadioGroup: React.FunctionComponent = () => {
   const [radioBorderWidth, setRadioBorderWidth] = React.useState<number>(1);
   const [radioVisibility, setRadioVisibility] = React.useState<number>(1);
   const [radioBorder, setRadioBorder] = React.useState<string>('black');
-  const [radioSize, setRadioSize] = React.useState<number>(20);
+  const [radioOuterCircleSize, setRadioOuterCircleSize] = React.useState<number>(20);
```

Comment:
There was a feedback to not show customisations as a demo in tester to prevent accidental usage promotions. Could you take a look at this https://github.com/microsoft/fluentui-react-native/pull/2444#discussion_r1054622829 - and follow similar changes for the tester?

Reply:
Android does not have the customizedRadioGroup, this change is related to the rename of the radioSize prop